sdk/spring/spring-cloud-azure-resourcemanager/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 4.4.0-beta.1 (Unreleased)
+## 4.4.0 (2022-08-08)
 
 ### Features Added
```

Comment:
Same here, and for any other `CHANGELOG.md` that doesn't contain the changes but links to the changelog file in `sdk/spring`.